### PR TITLE
fix: DH GitLab config

### PR DESCRIPTION
**Pull request by Roming22**
- [RHTAPBUGS-1300](https://issues.redhat.com//browse/RHTAPBUGS-1300)
- Remove duplicates

### CODE REVIEW

File: pkg/integrations/gitlab.go [hunk 1]
```diff
@@ -38,10 +38,10 @@ func (g *GitLabIntegration) PersistentFlags(p *pflag.FlagSet) {
...
 	p.StringVar(&g.host, "host", g.host,
 		"GitLab host, defaults to 'gitlab.com'")
-	p.StringVar(&g.clientId, "app-id", g.clientId,
-		"GitLab application client id")
-	p.StringVar(&g.clientSecret, "app-secret", g.clientSecret,
-		"GitLab application client secret")
+	// p.StringVar(&g.clientId, "app-id", g.clientId,
+	// 	"GitLab application client id")
+	// p.StringVar(&g.clientSecret, "app-secret", g.clientSecret,
+	// 	"GitLab application client secret")
```
Comment (otaviof) [inline pkg/integrations/gitlab.go]: Please remove these lines instead of just comment them out.

**Review by otaviof — CHANGES_REQUESTED** [review]
Just a minor change, please.

File: pkg/integrations/gitlab.go (hunk 1 above)
Comment (Roming22) [reply pkg/integrations/gitlab.go]: Done

**Review by otaviof — CHANGES_REQUESTED** [review]
By the way, could you also unwire the attributes here: 

https://github.com/redhat-appstudio/rhtap-cli/blob/70bd7e239842e095a473a2b37c9c50657bc4d19a/pkg/integrations/gitlab.go#L29-L30

**Comment by Roming22** [comment]
@otaviof I've now fully removed the feature.

**Pull request merged**